spec/pom.xml

```diff
@@ -72,6 +72,7 @@
                             <goal>process-asciidoc</goal>
                         </goals>
                         <configuration>
+                            <outputFile>${project.build.directory}/generated-docs/${project.build.finalName}.html</outputFile>
```

Comment:
Is this line needed? We defined `<outputFile>` twice.

Reply:
I suggest to define a property
<config-spec-html>${project.build.directory}/generated-docs/${project.build.finalName}.pdf</config-spec-html> so that the following places can refer to that property

Reply:
One if for the pdf and the other one is for html.

Reply:
I meant the line 77 also defines the element `<outputFile>`.

Reply:
Yes. I didn't notice you did a similar change and it got duplicated with the rebase.